.github/workflows/staging-deploy.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Staging Deploy
+
+on:
+  push:
+    branches:
+    - alpha_build_process
```

Comment:
Tested this, its working fine. You can change the branch to develop so that it can be merged in

Reply:
Done